client/src/modules/templates/bhIprScale.tmpl.html

```diff
@@ -0,0 +1,13 @@
+<div class="btn-group" uib-dropdown dropdown-append-to-body>
+  <a class="btn btn-default" uib-dropdown-toggle>
```

Comment:
If this is a dropdown, can you append the `bh-dropdown-auto-dropup` tag to this?  It will make sure it behaves correctly when at the bottom of the list.